src/vmm/tests/io_uring.rs

```diff
@@ -120,7 +120,7 @@ fn test_eventfd() {
     let user_data: u8 = 71;
     let buf = [0; 4];
     let epoll = Epoll::new().unwrap();
-    let mut ready_events = vec![EpollEvent::default(); 1];
+    let mut ready_events = [EpollEvent::default(); 1];
 
```

Comment:
nit: maybe reword the commit message to be less annoyance driven?

Reply:
Done :)